client/coral-admin/src/graphql/utils.js

```diff
@@ -66,21 +66,27 @@ function addCommentToQueue(root, queue, comment, sort) {
 
 function getCommentQueues(comment) {
   const queues = ['all'];
+
   if (comment.status === 'ACCEPTED') {
     queues.push('accepted');
   }
   else if (comment.status === 'REJECTED') {
     queues.push('rejected');
   }
-  else if (comment.status === 'PREMOD') {
-    queues.push('premod');
-  }
-  if (
+  else if (
     ['NONE', 'PREMOD'].indexOf(comment.status) >= 0
     && comment.actions && comment.actions.some((a) => a.__typename === 'FlagAction')
   ) {
     queues.push('flagged');
   }
+  else if (comment.status === 'PREMOD') {
+    queues.push('premod');
+    queues.push('new');
+  } 
+  if (comment.status === 'NONE') {
```

Comment:
This should be an `else if condition` and move up to join the other `else if` conditions (see explanation below)